.github/workflows/changeset-check.yml

```
@@ -0,0 +1,127 @@
+        with:
+          cwd: ./typescript
+          version: false
+          publish: false
+          setupGitUser: true
+          createGithubReleases: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Comment on PR if Python changes missing changeset
+        if: steps.filter.outputs.python == 'true' && steps.python-check.outputs.hasChangesets == 'false'
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)